src/index.js

```diff
@@ -92,7 +97,9 @@ export default class Routes extends Component {
                 render={() => (
                   <Request url={`/api/v1/instructors/${user.instructor_id}`}>
```

Comment:
Sounds good, I have a note to address upgrading off of meta in another PR.
![image](https://cloud.githubusercontent.com/assets/5497885/22611639/eafb83f4-ea28-11e6-848a-6710c37be1ef.png)